packages/plugin-input/src/InputNode.ts

```diff
@@ -0,0 +1,40 @@
+import { VElement } from '../../core/src/VNodes/VElement';
+
+export class InputNode extends VElement {
+    value: string;
+    constructor(type?: string, name?: string, value?: string) {
+        super({ htmlTag: 'INPUT' });
+        this.inputName = name;
+        this.inputType = type;
+        this.value = value;
+    }
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * Return the `name` attribute of this Input node.
+     */
+    get inputName(): string {
+        return this.attributes.name as string;
+    }
+    /**
+     * Set the `name` attribute of this Input node.
+     */
+    set inputName(name: string) {
+        this.attributes.name = name;
+    }
+    /**
+     * Return the `type` attribute of this Input node.
+     */
+    get inputType(): string {
+        return this.attributes.type as string;
+    }
+    /**
+     * Set the `type` attribute of this Input node.
+     */
+    set inputType(name: string) {
+        this.attributes.type = name;
+    }
```

Comment:
Attributes are a modifier that is specific to XML. If these values make sense for this node outside of the context of XML (I think it does) then I think they deserve a proper field on the node.

Reply:
I did the same here as on `LinkNode` with `url` though. Should I change both then?

Reply:
Well work on the modifiers changed the semantics of the attributes, so yes I suggest changing `LinkFormat` as well

Reply:
I should probably wait for `modifiers` though then...

Reply:
Alright, as long as you don't forget that you should not just change it to `modifiers.get(Attributes)` at that point and to change `LinkFormat`.